bundles/org.openhab.binding.opensprinkler/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -29,6 +29,14 @@
 				<description>Specifies the refresh interval in seconds.</description>
 				<default>60</default>
 			</parameter>
+			<parameter name="basicUsername" type="text">
```

Comment:
Is this an advanced configuration option? There are no clear rules but I think if less than 20% of the users need it, you should mark it as `<advanced>true</advanced>`. There is no difference for textual configuration but UIs can decide to hide it until full options are requested.

Reply:
This is absolutely an "advanced" feature, you need to already have a deeper knowledge about this stuff in order to setup the reverse proxy itself. I'll add the advanced thing :)